Code/Editor/EditorPreferencesPageViewportGizmo.cpp

```diff
@@ -28,6 +31,11 @@ void CEditorPreferencesPage_ViewportManipulator::Reflect(AZ::SerializeContext& s
                 AZ::Edit::UIHandlers::SpinBox, &Manipulators::m_manipulatorLineBoundWidth, "Line Bound Width",
                 "Manipulator Line Bound Width")
             ->Attribute(AZ::Edit::Attributes::Min, 0.001f)
+            ->Attribute(AZ::Edit::Attributes::Max, 2.0f)
```

Comment:
nit: for clarity I think its preffered to indent the Attributes like:

```
        ->DataElement(
                AZ::Edit::UIHandlers::SpinBox, &Manipulators::m_manipulatorLineBoundWidth, "Line Bound Width",
                "Manipulator Line Bound Width")
            ->Attribute(AZ::Edit::Attributes::Min, 0.001f)
            ->Attribute(AZ::Edit::Attributes::Max, 2.0f)
```

Reply:
This is now formatted with `clang-format` which doesn't do this but I still think it's very readable